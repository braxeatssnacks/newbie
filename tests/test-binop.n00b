
define function main with no params
	print("1 * 3")
	print(1*3)
	
	print("1 + 2")
	print(1+2)

	print("1 < 3")
	print(1<3)

	print("4 - 2")
	print(4-2)

	print("3 / 2")
	print(3/2)

	print("13 % 3")
	print(13%3)

	print("4 = 4")
	print(4=4)

	print("5 <= 7")
	print(5<=7)

	print("6 >= 6")
	print(6>=6)

	print("5 > 7")
	print(5>7)

	print("true and true")
	print(true and true)

	print("true or false")
	print(true or false)

	print("true and false")
	print(true and false)
