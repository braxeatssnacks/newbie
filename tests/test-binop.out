1 * 3
3
1 + 2
3
1 < 3
1
4 - 2
2
3 / 2
1
13 % 3
1
4 = 4
1
5 <= 7
1
6 >= 6
1
5 > 7
0
true and true
1
true or false
1
true and false
0